Bottom: 0461f67087e92a3cd52443eb488abbc73b66b926
Top:    38b9cfcc090133f3754133db19b267164b1d761c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-04 17:47:24 +0000

Avoid clearing pins / vias from polygons... it is already done!




---

diff --git a/src/draw.c b/src/draw.c
index 1d902b2..476e2cb 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1190,6 +1190,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1212,6 +1213,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
